vision/detect_crop_hints.rb

```diff
@@ -12,22 +12,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-def detect_crop_hints project_id:, image_path:
+require "uri"
+
+def detect_crop_hints image_path:
   # [START vision_crop_hint_detection]
-  # project_id = "Your Google Cloud project ID"
   # image_path = "Path to local image file, eg. './image.png'"
-  
+
   require "google/cloud/vision"
 
-  vision = Google::Cloud::Vision.new project: project_id
-  image  = vision.image image_path
+  image_annotator = Google::Cloud::Vision::ImageAnnotator.new
 
-  crop_hints = image.crop_hints
+  response = image_annotator.crop_hints_detection(
+    image: image_path
+  )
```

Comment:
Is there a reason this is broken out into multiple lines? It fits easily into 80 characters. (Same in a number of additional cases below.)